modules/sqs/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "env" {}
+
+variable "aws_profile" {}
+
+variable "enabled" {
+  default = true
+}
+variable "sqs_enabled" {
+  default = true
+}
+
+variable "dd_api_key" {
+  description = "Datadog API Key. You can find out how to get it here: https://docs.datadoghq.com/account_management/api-app-keys/"
+}
+
+variable "dd_app_key" {
+  description = "Datadog APP Key. You can find out how to get it here: https://docs.datadoghq.com/account_management/api-app-keys/"
+}
+
+variable "namespace" {}
+
+variable "service_name" {}
+
+# Threshold values
+
+variable "sqs_normal_value" {
+  description = ""
+  default = 0
+}
+
+variable "sqs_warning_value" {
+  description = ""
```

Comment:
Let's add a description

Reply:
@AutomationD Should I use "value" or "threshold" in sqs variables ? I mean in normal, warning and critical variables.

Reply:
If they are thresholds from the user standpoint, then use "threshold". So yes.